src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -161,7 +161,9 @@ export class CalciteDropdown {
             onTransitionEnd={this.transitionEnd}
             ref={this.setScrollerEl}
```

Comment:
This seems to work for me, I don't think we need another div element.

```
<div
            class={{
              ["calcite-dropdown-content"]: true,
              [PopperCSS.animation]: true,
              [PopperCSS.animationActive]: active
            }}
            hidden={!active}
            onTransitionEnd={this.transitionEnd}
            ref={this.setScrollerEl}
          >
```

Reply:
The issue with this approach is that `calciteDropOpen` event isn't emitted as the element is hidden.

Reply:
Gotcha. makes sense